rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/importexport/ImportExportController.java

```diff
@@ -198,4 +176,45 @@ public ResponseEntity<String> handleGlobalException(Exception e) {
         LOGGER.error("Unhandled exception: {}", e.getMessage(), e);
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(e.getMessage());
     }
+
+    @Operation(summary = "Upload component csv file.", description = "Upload component csv file.", tags = {
+            "Component" })
+    @RequestMapping(value = IMPORTEXPORT_URL + "/uploadComponent", method = RequestMethod.POST, consumes = {
+            MediaType.MULTIPART_MIXED_VALUE,
+            MediaType.MULTIPART_FORM_DATA_VALUE }, produces = { MediaType.APPLICATION_JSON_VALUE })
+    public ResponseEntity<RequestSummary> uploadComponentCsv(
+            @Parameter(description = "The component csv file to be uploaded.") @RequestParam("componentFile") MultipartFile file,
+            HttpServletRequest request, HttpServletResponse response) throws TException, IOException, ServletException {
+
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        RequestSummary requestSummary = importExportService.uploadComponent(sw360User, file, request, response);
+        return ResponseEntity.ok(requestSummary);
+    }
+
+    @Operation(summary = "release link file.", description = "release link file.", tags = { "Release" })
```

Comment:
The tag should be `ImportExport` as with other endpoints.

```suggestion
    @Operation(summary = "Import releases.", description = "Upload CSV with release link to import.", tags = { "ImportExport" })
```

Reply:
Also, please document the header needed as done in `downloadComponent()` for the `Content-Type` header (put in `consumes`).